pontoon/sync/changeset.py

```diff
@@ -259,6 +262,11 @@ def update_entity_translations_from_vcs(
             # Modify existing translation.
             if db_translation:
                 if not db_translation.approved and not vcs_translation.fuzzy:
+                    log_action(
+                        'translation:approved',
+                        user or self.sync_user,
+                        translation=db_translation,
+                    )
```

Comment:
We should use `bulk_create` instead (also for the `'translation:rejected'` action below).